fanuc.md

```diff
@@ -6,27 +6,54 @@ Fanuc controllers must have the following modules enabled:
 - R648 - User Socket Messaging
 
 Check the compatibility:
-1. Go to **MENU** -> **STATUS** -> **Version ID**.
-2. Click **ORDER FI**.
-3. Check for **R632** and **648** in the list.
+1. Go to **MENU** -> **NEXT** -> **STATUS** -> **Version ID**.
+![image](https://user-images.githubusercontent.com/67538561/169281756-e59e61af-3a40-4c8a-aa00-94123f3ba3e3.png)
+
+3. Click **ORDER FI**.
+![image](https://user-images.githubusercontent.com/67538561/169282210-865d139e-1bad-4598-80c0-086461c159a6.png)
+
+4. Check for **R632** and **648** in the list.
```

Comment:
It's rather unlikely `R632` is needed.

Controllers need to be able to execute Karel already, as that's what half the software Fanuc distributes is written in.

`R648` is definitely needed.